README.md

```diff
@@ -60,7 +60,7 @@ mkdir build
 cd build
 cmake ..
 make
-./test
+./test_abieos
```

Comment:
Probably should be `ctest` as there are other tests in addition to `test_abieos` (CI uses ctest)

Reply:
https://github.com/AntelopeIO/abieos/pull/47/commits/83c4a30cd7d22272eb86feed99f4e08ea4eb7530